### ISSUE TITLE
[Feature Request] Could we re-adjust alpha value after deleting points manually?

### ISSUE THREAD

**Issue by oUp2Uo** [hbb1/2d-gaussian-splatting#168]
Hi, thanks for the great work.

The main objects are great in result ply file.
But the backgound (eg. sky, far-away buildings, ...) is not very good since they are too far.
So sometimes there would be wrong points in front of/behind the main objects("frogs").
If we try to delete the wrong points, the main objects would be transparent in some part.
Could we de some post process to re-adjust alpha value to fix these transparent parts?
Or if we can train from gs ply file, not normal ply file, then we could only adjust alpha just using the `train.py` by setting some parameters like `densify_until_iter=0`, to adjust alpha only.

Thank you.

**Comment by hbb1** [comment]
I completely agree. I think it is worth investigating. It is possible to fine-tune the appearance while fixing geometry. I also believe there may be more effective way to handle densitification. Note our densification is different than 3DGS and it may add fewer points to distant regions.

**Comment by oUp2Uo** [comment]
> I completely agree. I think it is worth investigating. It is possible to fine-tune the appearance while fixing geometry.

Thanks for the reply.

I have tried to search papers, but could not find some about this point.
I thought post-processing (eg. re-adjust alpha / fix hole / smooth surface / ...) is very important for GS to be used like mesh.
And mask for the source images may be another way to avoid the problem. If using mask, we could do not generate the far away points.

**Comment by oUp2Uo** [comment]
I have managed to re-train by only adjusting alpha.
The result seems like to put a "patch" on the surface, which surely fixs transparent parts, but not very good.
![1](https://github.com/user-attachments/assets/fed07e4d-96bb-4134-8a73-dab9a83aea66)
If we want to fix the holes, which is generated by manually deleting points, I think not only re-adjust alpha, but also need to add points.
